frontend/app/route-helpers/apply-route-helpers.server.ts

```diff
@@ -58,6 +58,16 @@ export interface ApplyState {
   readonly dentalInsurance?: boolean;
   readonly disabilityTaxCredit?: boolean;
   readonly livingIndependently?: boolean;
+  readonly parentGuardianInformation?: {
+    firstName: string;
+    lastName: string;
+    maritalStatus: string;
+    socialInsuranceNumber: string;
+    dateOfBirthYear: number;
+    dateOfBirthMonth: number;
+    dateOfBirthDay: number;
+    dateOfBirth: string;
+  };
```

Comment:
This is not needed since we already have `applicantInformation` and `dateOfBirth` to save this information. Otherwise, the data that will be saved here cannot be reused in other flows if the client descide to change the type of application. Please remove it and use `applicantInformation` and `dateOfBirth`.